UT4MasterServer/Controllers/SessionController.cs

```diff
@@ -148,7 +148,10 @@ public class SessionController : JsonAPIController
 
 		if (account == null)
 			account = await accountService.GetAccountAsync(session.AccountID);
-		logger.LogInformation($"User {account} was authorized via {grantType}");
+		/* TODO: Account still can be null if session.AccountID is empty GUID, for example, if we are using access_token from client_credentials grant type.
+		 * What to do here?
+		 */
+		logger.LogInformation($"User '{account}' was authorized via {grantType}");
```

Comment:
I don't think there's an issue with `account` being `null`. If it's null we just don't print user related info in response. Json generating code below does account null checking.

For the sake of logging there should be `account ?? EpicID.Empty.ToString()` in logged string.

Reply:
It gives me `Operator '??' cannot be applied to operands of type 'UT4MasterSever.Models.Account?' and 'UT4MasterServer.Other.EpicID'?`, so I just deleted TODO

Reply:
Then i suppose instead of  `account ?? EpicID.Empty` use `account?.ToString() ?? EpicID.Empty.ToString()`

Reply:
Done